app/controllers/api/users_controller.rb

```diff
@@ -12,24 +12,15 @@ def index
 
     @target = target_allowlist.include?(params[:target]) ? params[:target] : 'student_and_trainee'
 
-    target_users =
-      if @target == 'followings'
-        current_user.followees_list(watch: @watch)
-      elsif @tag
-        User.tagged_with(@tag)
-      elsif @company
-        User.where(company_id: @company).users_role(@target)
-      elsif @target.in? %w[hibernated retired]
-        User.users_role(@target)
-      else
-        User.users_role(@target).unhibernated.unretired
-      end
+    users = target_users
+    users.order(:last_activity_at) if @target == 'inactive'
```

Comment:
```suggestion
    users = users.order(:last_activity_at) if @target == 'inactive'
```

としないとorderが反映されないのでは

Reply:
すみません、`order`は破壊的な変更ではないですね！勘違しておりました。
変数に代入し直す様に修正しました。
6cd6a45f4fa5407a586a816dd3d28138244b6cbe